google/datalab/bigquery/_table.py

```diff
@@ -388,7 +388,7 @@ def _init_job_from_response(self, response):
       job = _job.Job(job_id=response['jobReference']['jobId'], context=self._context)
     return job
 
-  def extract_async(self, destination, format='csv', csv_delimiter=',', csv_header=True,
+  def extract_async(self, destination, format='csv', csv_delimiter=None, csv_header=True,
```

Comment:
I am thinking would it be better if we set it to None if format != 'csv'? Setting default to None seems not clear (docstring says defaults to ',', but seems not true from the function interface).